workflow/scripts/generate_pindel_config.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
I have added feature request for adding logging and tests to this script

Reply:
I am working on the tests, logging will be added as well 👍